recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
@bgruening Thanks but I cannot find any information about the `$PREFIX` issue. Anyway, I edited my `build.sh` and `test.sh` scripts and now their 1st line is `cd $PREFIX/bin/`
I hope this satisfies the requirements.